data/core/units/dunefolk/Apothecary.cfg

```diff
@@ -8,40 +8,45 @@
     hitpoints=39
     movement_type=dunefoot
     [abilities]
-        {ABILITY_CURES}
-        {ABILITY_REGENERATES}
+        {ABILITY_EXTRA_HEAL}
+        {ABILITY_SELF_HEAL}
     [/abilities]
     movement=5
     experience=100
     level=2
     alignment=liminal
-    advances_to=null
-    {AMLA_DEFAULT}
+    advances_to=Dune Luminary
     cost=33
     usage=fighter
-    description= _ "Some men excel at medicine the way others do at strength of arms, and the Apothecaries are those whose knowledge is without peer outside of the Dunefolk. Their remedies can not only heal wounds but remove infection and poison as well, often to the disbelief of the injured. Hardened by years of travels, many are able to treat themselves."+{SPECIAL_NOTES}+{SPECIAL_NOTES_CURES}+{SPECIAL_NOTES_REGENERATES}
-    die_sound={SOUND_LIST:ELF_HIT}
-    {DEFENSE_ANIM "units/dunefolk/apothecary.png" "units/dunefolk/apothecary.png" {SOUND_LIST:ELF_HIT} }
+    description= _ "Problems: Current Apothecary says  [Hardened by years of travels, many are able to treat themselves.] This sentence could be now applied in a way on the Lv1 Herbalist since they too can heal themselves. Considerable to just simple move this line from Lv2 Apothecary to Lv1 Herbalist.
```

Comment:
Looks like comment

Reply:
Also needs resolution.

Reply:
I've got no idea why the sounds are for elf. Here in file is defined
{SOUND_LIST:HUMAN_HIT} and {SOUND_LIST:HUMAN_DIE}.
https://github.com/FranPrin/wesnoth/blob/master/data/core/units/dunefolk/Apothecary.cfg

Reply:
Yes, it is a comment, but I suppose Yumi (nemaara) will have to do the description for Apothecary, or use the default with some changes.